.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,29 @@
+#### Is this a Bug or Feature request?:
+
+
+#### Steps to reproduce or link to a repro project:
+
+
+#### Expected  behavior:
+
+
+#### Actual behavior:
+
+
+#### Environment Information:
+- **TargetFramework (can be found in .csproj file):**
+
+
+- **Version of `Microsoft.AspNetCore.Mvc` or `Microsoft.AspNetCore.App` or `Microsoft.AspNetCore.All`:**
```

Comment:
Beyond the version of MVC this information is almost never useful.  I think it can be omitted except for the ASP.NET Core version